From 0d1548edd4994385c4de41a3fdfcc3eedf73cb5a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:55 -0400
Subject: [redhat] kabi: Adding symbol kmalloc_caches (mm/slab_common.c)

Message-id: <e09a45ed8f2b63d8eac8700b3ff987f8d68aa9f1.1410510773.git.poros@redhat.com>
Patchwork-id: 96094
O-Subject: [PATCH RHEL7.1 115/233] kabi: Adding symbol kmalloc_caches (mm/slab_common.c)
Bugzilla: 1062361 1077335
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361,1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Through use of kmalloc.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/kmalloc_caches
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/kmalloc_caches
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/kmalloc_caches

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_caches b/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_caches
new file mode 100644
index 0000000..24f1559
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/kmalloc_caches
@@ -0,0 +1,2 @@
+#1-
+0x1c8a8817 kmalloc_caches vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/kmalloc_caches b/redhat/kabi/kabi-module/kabi_s390x/kmalloc_caches
new file mode 100644
index 0000000..92c69a9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/kmalloc_caches
@@ -0,0 +1,2 @@
+#1-
+0x646af8b4 kmalloc_caches vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_caches b/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_caches
new file mode 100644
index 0000000..a1beb7a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/kmalloc_caches
@@ -0,0 +1,2 @@
+#1-
+0x98ab5c8d kmalloc_caches vmlinux EXPORT_SYMBOL
-- 
1.7.1